vast/vast.cpp

```diff
@@ -164,43 +165,47 @@ int main(int argc, char** argv) {
     // success when printing the help/documentation texts.
     return EXIT_SUCCESS;
   }
-  // Initialize actor system (and thereby CAF's logger).
-  if (!init_config(cfg, *invocation, std::cerr))
+
+  // since the logger is not dependend on caf, this could be moved up
```

Comment:
Please correct the comment and move this to as early as possible in this function, with a comment explaining why this is the earliest possible point.